packages/frontend/src/components/DashboardTiles/TileBase/index.tsx

```diff
@@ -59,16 +59,20 @@ const TileBase = <T extends Dashboard['tiles'][number]>({
         tile.type !== DashboardTileTypes.MARKDOWN
             ? tile.properties.hideTitle
             : false;
+
+    const hasTitle = hideTitle ? false : true;
     return (
         <TileBaseWrapper
             className={isLoading ? Classes.SKELETON : undefined}
             isEditMode={isEditMode}
             isHovering={isHovering}
+            hasTitle={hasTitle}
         >
             <HeaderContainer
                 isEditMode={isEditMode}
                 onMouseEnter={() => setIsHovering(true)}
                 onMouseLeave={() => setIsHovering(false)}
+                hasTitle={hasTitle}
```

Comment:
you can just `!hideTitle`
was little bit confusing to see `hideTitle ? false : true`